### PR TITLE
oopsy: don't report dead players for missed buffs

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/oopsyraidsy/effect_tracker.ts [hunk 1]
```diff
@@ -266,7 +294,13 @@ export class EffectTracker {
     for (const id of collected.targetIds)
       gotBuffMap[id] = true;
 
-    const missedIds = this.partyTracker.partyIds.filter((id) => !gotBuffMap[id] && id !== emptyId);
+    const missedIds = this.partyTracker.partyIds.filter((id) => {
+      // Filter out any empty ids here.
+      if (id === emptyId)
+        return false;
```
Comment (panicstevenson) [inline ui/oopsyraidsy/effect_tracker.ts:300]: This is unrelated to the dead people thing, right?

**Review by panicstevenson — APPROVED** [review]

File: ui/oopsyraidsy/effect_tracker.ts (hunk 1 above)
Comment (quisquous) [reply ui/oopsyraidsy/effect_tracker.ts:300]: Yeah.  This check was there before.

Comment (panicstevenson) [reply ui/oopsyraidsy/effect_tracker.ts:300]: Oh, I'm bad at reading. I read the two checks as the same two checks below. Obviously that doesn't make sense.
